options/options.cc

```diff
@@ -516,7 +516,8 @@ ColumnFamilyOptions* ColumnFamilyOptions::OptimizeLevelStyleCompaction(
     if (i < 2) {
       compression_per_level[i] = kNoCompression;
     } else {
-      compression_per_level[i] = kSnappyCompression;
+      compression_per_level[i] =
+          Snappy_Supported() ? kSnappyCompression : kNoCompression;
```

Comment:
I prefer to make this change in a separate PR if possible. In my mind, unity test fix would be a much lower priority than this fix ... so I don't want it to be buried as part of a unity test fix when someone "blames" the code later.

Reply:
will submit a separate PR for that, thanks